src/pages/veiligheidsregio/[code]/verpleeghuiszorg.tsx

```diff
@@ -0,0 +1,199 @@
+import CoronaVirus from '~/assets/coronavirus.svg';
+import Locatie from '~/assets/locaties.svg';
+import Getest from '~/assets/test.svg';
+import { KpiTile } from '~/components-styled/kpi-tile';
+import { KpiValue } from '~/components-styled/kpi-value';
+import { LineChartTile } from '~/components-styled/line-chart-tile';
+import { TwoKpiSection } from '~/components-styled/two-kpi-section';
+import { Text } from '~/components-styled/typography';
+import { ContentHeader } from '~/components/contentHeader';
+import { FCWithLayout } from '~/components/layout';
+import { getSafetyRegionLayout } from '~/components/layout/SafetyRegionLayout';
+import { SEOHead } from '~/components/seoHead';
+import siteText from '~/locale/index';
+import {
+  getSafetyRegionData,
+  getSafetyRegionPaths,
+  ISafetyRegionData,
+} from '~/static-props/safetyregion-data';
+import { replaceVariablesInText } from '~/utils/replaceVariablesInText';
+
+const locationsText = siteText.veiligheidsregio_verpleeghuis_besmette_locaties;
+const positiveTestPeopleText =
+  siteText.veiligheidsregio_verpleeghuis_positief_geteste_personen;
+const mortalityText = siteText.veiligheidsregio_verpleeghuis_oversterfte;
+
+const NursingHomeInfectedLocations: FCWithLayout<ISafetyRegionData> = (
+  props
+) => {
+  const { data, safetyRegionName } = props;
+
+  const newlyInfectedLocations =
+    data.nursing_home.last_value.newly_infected_locations;
+  const infectedLocationsTotal =
+    data.nursing_home.last_value.infected_locations_total;
+  const infectedLocationsPercentage =
+    data.nursing_home.last_value.infected_locations_percentage;
```

Comment:
If you do this
```suggestion
    const nursingHomeLastValue =
    data.nursing_home.last_value
```

And then index the data you need inline, I believe the code will be a bit more readable and you can reuse it in more places as well.

Reply:
You could also just call it `nursingHome` to make it more terse. I don't think it is too vague

Reply:
or simply `lastValue`. The context "verpleeghuiszorg" is clear enough imho.

Reply:
Fixed